lib/full_text_search/journal_mapper.rb

```diff
@@ -33,7 +33,9 @@ def upsert_fts_target(options={})
       return if issue.nil?
       return unless issue.is_a?(Issue)
 
-      fts_target = find_fts_target
+      fts_target = find_fts_target(initialize: options.fetch(:initialize, true))
+      return if fts_target.blank?
```

Comment:
The `find_by` returns `nil`.
ref: https://api.rubyonrails.org/v8.0.2/classes/ActiveRecord/FinderMethods.html#method-i-find_by
```suggestion
      return unless fts_target
```